spec/controllers/recipe_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../rails_helper'
+RSpec.describe 'Recipe', type: :request do
+  context 'should have status ok 200' do
+    it 'index action should have status ok' do
+      get '/users/1/recipes'
+      expect(response).to have_http_status(:ok)
+    end
+    it 'show action should have status ok' do
+      get '/users/1/recipes/1'
+      expect(response).to have_http_status(:ok)
+    end
+  end
+  context 'should render_template' do
+    it 'index action should render index template' do
+      get '/users/1/recipes'
+      expect(response).to render_template('index')
+    end
+    it 'show action to render template show' do
+      get '/users/1/recipes/1'
+      expect(response).to render_template('show')
+    end
+  end
```

Comment:
- [ ] For some reason, the following test is failing. Below is the error I am getting in the console. Kindly look into why this is the case.

![image](https://user-images.githubusercontent.com/86318284/219387568-ae206da3-d799-4810-accd-03c2a80b97bc.png)